submissions/lillianNwaoha/css/signup.css

```diff
@@ -0,0 +1,320 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  overflow-x: hidden;
+}
+nav {
+  background-color: #0066f5;
+  height: 190px;
+  width: auto;
+}
+.logo img {
+  width: 90px;
```

Comment:
Only set either of the height or width not both @Lillian-1

Reply:
Kindly fix this comment @ToluAkin dropped of your PR

Reply:
I have fixed this code